src/settings/NoticePolicy/components/EditSections/FeeFineNoticesSection/FeeFineNoticesSection.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { FieldArray } from 'react-final-form-arrays';
+
+import { Accordion } from '@folio/stripes/components';
+import FeeFineNoticesSection from './FeeFineNoticesSection';
+import NoticesList from '../components';
+
+import {
+  feeFineNoticesTriggeringEvents,
+  timeBasedFeeFineEventsIds,
+  uponAndAfterSendEvents,
+} from '../../../../../constants';
+
+const mockedCatchFunc = jest.fn();
+
+jest.mock('react-final-form-arrays', () => ({
+  FieldArray: jest.fn(({ getSendEvents }) => {
+    mockedCatchFunc(getSendEvents());
```

Comment:
could you please move `getSendEvents` to the top and export it and test it?

Reply:
done